dependencyci.yml

```diff
@@ -1,6 +1,4 @@
 # don't run unmaintained test on any dependencies in Gemfile
```

Comment:
Is this comment out of date now?  (since we don't specifically call out the `Gemfile` in the configuration...)